### PR TITLE
Flower wreaths

### PR DESCRIPTION
**Pull request by UbaserB**
## About the PR

Flower wreaths are a craftable neck accessory with the same recipe as flower crowns.

## Media

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

image

image

**Changelog**

:cl: Ubaser
- add: You can now craft flower wreaths.

### CODE REVIEW

File: Resources/Prototypes/Recipes/Crafting/Graphs/improvised/flowerwreath.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+                sprite: Objects/Specific/Hydroponics/poppy.rsi
+                state: produce
+            - tag: Ambrosia
+              name: ambrosia
+              icon:
+                sprite: Objects/Specific/Hydroponics/ambrosia_vulgaris.rsi
+                state: produce
+              doAfter: 10
+    - node: flowerwreath
+      entity: ClothingHeadHatFlowerWreath
```
Comment (Emisse) [inline Resources/Prototypes/Recipes/Crafting/Graphs/improvised/flowerwreath.yml:26]: i thought this was ClothingNeckFlowerWreath

Comment (UbaserB) [reply Resources/Prototypes/Recipes/Crafting/Graphs/improvised/flowerwreath.yml:26]: oh yeah my mistake when i coped it over

**Comment by lzk228** [comment]
maybe would be better to not bloat items, but make flower crown equipable to neck?

**Comment by UbaserB** [comment]
in that case the flower crown would have to be renamed but thats for another pr

**Comment by Aexxie** [comment]
> in that case the flower crown would have to be renamed but thats for another pr

Flower wreath fits both of those, I kinda like the idea of it being able to be used in both ways.

**Pull request merged**